pkgs/tools/cd-dvd/unetbootin/default.nix

```diff
@@ -1,13 +1,13 @@
-{ stdenv, fetchurl, makeWrapper, qt4, utillinux, coreutils, which, p7zip, mtools, syslinux }:
+{ stdenv, fetchurl, makeWrapper, qt4, utillinux, coreutils, which
+, p7zip, mtools, syslinux }:
 
-let version = "608"; in
-
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   name = "unetbootin-${version}";
+  version = "613";
 
```

Comment:
Oh. If that's the only reason, you can just write `let version = ...; in` on one line...

Looks good otherwise. I just succesfully booted an installer made with this.